src/cgame/cg_rocket_datasource.cpp

```diff
@@ -1653,11 +1653,9 @@ void CG_Rocket_BuildAlienEvolveList( const char *table )
 			}
 
 			buf[ 0 ] = '\0';
-			price = BG_CostToEvolve( cg.predictedPlayerState.stats[ STAT_CLASS ], i );
-			if( price == CANT_EVOLVE){
-				price = 0;
-			}
-			if( price < 0 ){
+			price = static_cast<float>( BG_ClassEvolveInfoFromTo(
+						cg.predictedPlayerState.stats[ STAT_CLASS ], i ).evolveCost );
+			if( price < 0.0f ){
```

Comment:
Nit: also check that the class is unlocked (according to the evolveInfo_t) instead of assuming the value is nonnegative in that case?

Reply:
seems fine to me. Let's say I'm a tyrant and that we lost a lot of momentum so that adv goon is locked again. It would still make sense to say that I will recover 1.8 morph point if I devolve to adv goon.